subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -39,14 +45,19 @@
 #include "common/log.h"
 #include "hal/debug.h"
 
-uint8_t ll_cis_accept(uint16_t handle)
+static struct ll_conn *ll_cis_reply_ok(uint16_t handle, uint8_t *error)
```

Comment:
Function name implies a boolean return value, but that's not the case here. Suggest to rename

Reply:
Why isn't it? `ll_cis_reply_ok` is added by this PR, isn't it?

Reply:
Ah, now I see what you mean. Will rename